linera-views/src/scylla_db.rs

```diff
@@ -205,11 +206,8 @@ impl ScyllaDbClientInternal {
     ) -> Result<(), ScyllaDbContextError> {
         let session = &client.0;
         let table_name = &client.1;
-        let values = (table_name.to_string(), key);
-        let query = format!(
-            "DELETE FROM kv.pairs_{} WHERE table_name = ? AND k = ?",
-            table_name
-        );
+        let values = (key,);
+        let query = format!("DELETE FROM kv.p_{} WHERE dummy = 0 AND k = ?", table_name);
```

Comment:
Why do we need the `dummy` column?

Reply:
Because it is required for comparison. Otherwise, it does not pass.